designsystem.digdir.no/drafts/UpdateLayout/UpdateLayout.module.css

```diff
@@ -0,0 +1,90 @@
+.content {
+    margin-top: 48px;
+    margin-bottom: 48px;
+}
+
+.title {
+    font-size: 40px;
+    margin-bottom: 24px;
+}
+
+.desc {
+    font-size: 18px;
+    line-height: 1.6em;
+    margin-bottom: -10px;
+}
+
+.left {
+
+}
+
+.right {
+
+}
```

Comment:
Should we just delete those classes? Should be pretty forward to add them again if needed. :)